src/GafferUI/StandardNodeGadget.cpp

```diff
@@ -548,12 +872,12 @@ LinearContainer *StandardNodeGadget::paddingRow()
 {
 	if( m_auxiliary )
 	{
-		return getChild<Gadget>( 0 ) // contentsColumn
+		return getChild<Gadget>( 1 ) // contentsColumn
 			->getChild<LinearContainer>( 1 );
 	}
 	else
 	{
-		return getChild<Gadget>( 0 ) // column
+		return getChild<Gadget>( 1 ) // column
```

Comment:
I kind of hate this code, especially now that FocusGadget adds itself as a child, so searching for addChild in StandardNodeGadget won't even reveal why `column` isn't the first child.

Reply:
Yeah, that and the `m_auxiliary` stuff have definitely made the internals of StandardNodeGadget harder to understand. I'm hopeful we can refactor our way out of that in the future though...